tests/migration/register_origin_system.pm

```diff
@@ -0,0 +1,57 @@
+# SLE15 migration tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Register the original system before migraiton
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
```

Comment:
what are the essential lines we need from this file to make a simple migration work, the one single case you picked in this case?
`register_system_in_textmode` and something else?
Let's just start by the most simple code and adding more once we need it, so we can keep this refactors healthy.